texttospeech/snippets/synthesize_speech_multiple_speakers.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Google Cloud Text-To-Speech API sample application .
+
+Example usage:
+    python synthesize_speech_multiple_speakers.py
+"""
+
+
+def synthesize_speech_multiple_speakers():
+    # [START tts_synthesize_speech_multiple_speakers]
+    """Synthesizes speech for multiple speakers.
+    Make sure to be working in a virtual environment.
+    """
+    from google.cloud import texttospeech_v1beta1 as texttospeech
+
+    # Instantiates a client
+    client = texttospeech.TextToSpeechClient()
+
+    multi_speaker_markup = texttospeech.MultiSpeakerMarkup(
+        turns=[
+            texttospeech.MultiSpeakerMarkup.Turn(
```

Comment:
Just a thought! 

Why not do `from google.cloud import texttospeech_v1beta1.MultiSpeakerMarkup.Turn as Turn`
texttospeech. ?

Reply:
Because our official styles say to not import individual classes. (Even though I disagree for samples)